test/e2e/ccm_test.go

```diff
@@ -22,15 +22,15 @@ package e2e
 import (
 	"context"
 
-	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	"github.com/onsi/gomega/gstruct"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/cluster-api/test/framework/clusterctl"
 )
 
-var _ = Describe("Nutanix flavor CCM [PR-Blocking]", func() {
+var _ = Describe("Nutanix flavor CCM", func() {
```

Comment:
same

Reply:
this is definitely not pr blocking as ccm is not yet released